src/contexts/StateContextProvider.tsx

```diff
@@ -95,6 +97,8 @@ const STATE_DEFAULT: Readonly<StateContextType> = Object.freeze({
     startQuery: () => null,
 });
 
+const INITIAL_QUERY_PROGRESS = 0;
```

Comment:
Do you feel this const would make more sense to sit below `type QueryResults = Map<number, QueryResultsType[]>;` in `src/typings/worker.ts`?